hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/SynthEventService.java

```diff
@@ -0,0 +1,47 @@
+package com.hedera.mirror.importer.domain;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
```

Comment:
nit: here and in any new file
```suggestion
 * Copyright (C) 2019 - 2023 Hedera Hashgraph, LLC
```

Reply:
Fixed.